src/components/Playback/Playback.tsx

```diff
@@ -37,7 +37,9 @@ export class Playback extends Component<Props, State> {
     }
 
     setMedia(media: { blob: Blob; mimeType: string }) {
-        const { autoPlay, pictureInPicture } = this.props;
+        const { pictureInPicture } = this.props;
```

Comment:
@odedm-kaltura shouldn't you delete the autoplay prop from the interface and from all the places that call this function?
it's redundant now

Reply:
this is not a talk for the component. 
it should be handled by the places using it.